peer/brontide_test.go

```diff
@@ -48,11 +48,14 @@ func TestPeerChannelClosureShutdownResponseLinkRemoved(t *testing.T) {
 
 	mockSwitch := &mockMessageSwitch{}
 
-	alicePeer, bobChan, err := createTestPeerWithChannel(
+	returnParams, err := createTestPeerWithChannel(
```

Comment:
nit: typically we call this `harness` in other places where this pattern is used